index.bs

```diff
@@ -2924,12 +2924,12 @@ Advancement on the Recommendation Track</h4>
 	For later stages,
 	especially transition to [=Candidate Recommendation|Candidate=] or [=Proposed Recommendation=],
 	there is usually a formal review meeting
-	to ensure the requirements have been met before [=Director=]'s approval is given.
+	to ensure the requirements have been met before approval is given.
 
 	[=Transition Requests=] to [=First Public Working Draft=]
 	or [=Candidate Recommendation=]
 	will not normally be approved
-	while a [=Working Group=]'s [=charter=] is undergoing or awaiting a [=Director=]'s decision
+	while a [=Working Group=]'s [=charter=] is undergoing or awaiting a decision
 	on an [=Advisory Committee Review=].</p>
```

Comment:
s/on an AC Review/after an AC Review/ for clarity

Reply:
I think I prefer it the way it is, but I don't feel strongly. Any third opinion?

Reply:
agree to leave it unchanged